core/store/orm/orm.go

```diff
@@ -1298,13 +1298,17 @@ func (orm *ORM) FindLogConsumer(lc *models.LogConsumption) (models.JobSpec, erro
 	return orm.FindJob(lc.JobID)
 }
 
+// FindOrCreateFluxMonitorRoundStats find the round stats record for agiven oracle on a given round, or creates
+// it if no record exists
 func (orm *ORM) FindOrCreateFluxMonitorRoundStats(aggregator common.Address, roundID uint32) (models.FluxMonitorRoundStats, error) {
 	orm.MustEnsureAdvisoryLock()
 	var stats models.FluxMonitorRoundStats
 	err := orm.db.FirstOrCreate(&stats, models.FluxMonitorRoundStats{Aggregator: aggregator, RoundID: roundID}).Error
```

Comment:
Orthoganal to this PR but I'd really like to see us moving towards upserts rather than this racy `FirstOrCreate` pattern.